h2o-genmodel/src/main/java/hex/genmodel/algos/gam/GamMojoModel.java

```diff
@@ -0,0 +1,37 @@
+package hex.genmodel.algos.gam;
+
+public class GamMojoModel extends GamMojoModelBase {
+  String _link;
+  double _tweedieLinkPower;
+  private boolean _binomial;
+  
+  GamMojoModel(String[] columns, String[][] domains, String responseColumn) {
+    super(columns, domains, responseColumn);
+  }
+  
+  void init() {
+    super.init();
+    _binomial = _family.equals("binomial") || _family.equals("fractionalbinomial") || _family.equals("quasibinomial");
```

Comment:
using String comparison here sounds dangerous and error prone, ohter Mojos use DistributionFamily internally as well, I suggest chaning the reader to do DistributionFamily.valueOf and use DistributionFamily internally, same for _link